dotnet3.5/S3/ListObjectsExample/ListObjectsTest/ListObjectsTest.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+using Amazon.S3;
+using Amazon.S3.Model;
+using Moq;
+using System;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ListObjectsTest
+{
+    public class ListObjectsTest
+    {
+        // This application uses XUnit and moq to test the CreateBucket Example.
```

Comment:
This comment seems unnecessary -- it seems obvious by scanning the using statements above.

Reply:
removed